platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/problems/ConfigurationCacheProblems.kt

```diff
@@ -138,14 +138,35 @@ class ConfigurationCacheProblems(
         return "configuration-cache"
     }
 
+    fun reportFailure(summary: Summary = summarizer.get(), htmlReportFile: File? = null): Throwable? {
+        val failDueToProblems = summary.failureCount > 0 && isFailOnProblems
+        val hasTooManyProblems = hasTooManyProblems(summary)
+        val cacheActionText = cacheAction.summaryText()
```

Comment:
💅 This should be inlined since it is not used in the `else` branch below

Reply:
Done.